media-video/projectx/projectx-0.91.0.10-r5.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit desktop java-pkg-2 java-pkg-simple toolchain-funcs xdg
+
+XDG_P="xdg-20100731"
+
+DESCRIPTION="Converts, splits and demuxes DVB and other MPEG recordings"
+HOMEPAGE="https://project-x.sourceforge.net/"
+SRC_URI="https://dev.gentoo.org/~billie/distfiles/${PF}.tar.xz
```

Comment:
Please replace ${PF} with ${P}-r4.
Here I abused the gentoo revision. ;-)
The package is not really developed and the original tarball was messy. So I cleaned it up and put it in my dev space with that name.

Reply:
Done.